modules/ROOT/pages/dataweave-flow-control.adoc

```diff
@@ -103,13 +104,75 @@ if (myVar.country == "USA")
 else { currency: "EUR" }
 ----
 
-.Output
+Note that you can use `if else` statements on any conditions that evaluate to `true` or `false`, including mathematical, logical, equality, and relational
+conditions. For example, you can replace `if (myVar.country == "USA")` in the script with `if ( 1 + 1 >= 2)`. 
+
+.Output:
 ----
 {
   "currency": "EUR"
 }
 ----
 
+
+An `if else` statement can act on any valid DataWeave input and return the data in any valid output format. For example, the following script transforms the input to XML and uses `if else` statements within the following:
+
+* A `using` statement that accompanies a `map` operation on the `firstInput` value
+* A mathematical operation in `ex1` that can return `true` or `false`
+* An equality operation in `ex2` that checks the index of an array and returns a string if the result is not `1`
+* An operation in `ex3` that either returns the `id` defined by the `using` statement or the specified string
+* A check in `ex4` that returns the string `"not null"` or `"null value"` depending on whether the value of the `id` is `null`
+
+
+.DataWeave Script:
+[source,dataweave,linenums]
+----
+%dw 2.0
+var firstInput = [
+  { "bookId":"101",
+    "title":"world history",
+    "price":"19.99"
+  },
+  {
+    "bookId":"202",
+    "title":"the great outdoors",
+    "price":"15.99"
+  }
+]
+output application/xml 
+---
+xmlRoot: { xmlChild : 
+	firstInput map (value) -> using (id = (if (value.bookId as Number == 202) "the book ID really is 202" else value.bookId as Number))
```

Comment:
Please use do instead of using

Reply:
do example from you is in else-if section

added another example in if-else that removes using but shows a variety of conditions